docs/node-operators/how-to/upgrade-network.md

```diff
@@ -7,57 +7,42 @@ hide_title: false
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-The protocol upgrade feature allows the nodes running a network to automatically update to the latest version of the Vega protocol, without requiring manual intervention. This can be particularly useful in situations where the protocol is critical to the functioning of the system and any downtime or disruption needs to be minimised.
+The Protocol Upgrade feature allows the Node Operators synchronously update nodes to the next version of the Vega Protocol.
```

Comment:
should this not be "asynchronously"?

Reply:
Actually they I think in this case it is meant `synchronously`, because they upgrade will take place synchronously but automatically.

Reply:
So maybe "automatically" instead of synchronously?